src/sickle.ts

```diff
@@ -68,7 +68,7 @@ export function getJSDocAnnotation(comment: string): JSDocComment {
     let match = line.match(/^@(\S+) *(.*)/);
     if (match) {
       let [_, tagName, text] = match;
-      if (text[0] == '{') {
+      if (['param', 'return', 'type'].indexOf(tagName) != -1 && text[0] == '{') {
```

Comment:
Yeah, I changed the error message in the follow up CL (I handle @type specially first).